features/checkout/seeing_order_summary/seeing_prices_recalculated_by_exchange_rate_on_order_summary.feature

```diff
@@ -0,0 +1,32 @@
+@checkout
+Feature: Seeing prices recalculated by exchange rate on order summary
+    In order to be certain about order cost in selected currency
+    As a Customer
+    I want to see my order summary with prices recalculated by exchange rate
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has currency "Zloty" with exchange rate 3.0
+        And the store has "VAT" tax rate of 23% for "Clothes" within "FR" zone
+        And the store has a product "Lannister Coat" priced at "$100.00"
+        And it belongs to "Clothes" tax category
+        And there is a promotion "Christmas sale"
+        And this promotion gives "10%" off on every product with minimum price at "$50.00"
+        And the store has currency Zloty with exchange rate 3.0
+        And the store has "DHL" shipping method with "$20.00" fee
+        And the store allows paying offline
+        And I am a logged in customer
+
+    @todo
+    Scenario: Seeing prices reclaculated by exchange rate from given currency
+        Given I change my currency to "Zloty"
+        And I add the product "Lannister Coat" to the cart
+        And I specified the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I proceed order with "DHL" shipping method and "Offline" payment
+        Then I should be on the checkout summary step
+        And the "Lannister Coat" product should have unit price "PLN270.00"
```

Comment:
Formatting looks weird (still en_US locale, not pl_PL), can we use british pounds or something like that instead? :)